src/SFA.DAS.EmployerApprenticeshipsService.Web.UnitTests/Orchestrators/Mappers/CommitmentMapperTests/WhenMappingToTransferCommitmentViewModel.cs

```diff
@@ -30,6 +31,8 @@ public void Arrange()
         {
             _hashingService = new Mock<IHashingService>();
             _hashingService.Setup(x => x.HashValue(It.IsAny<long>())).Returns((long p) => $"XYZ{p}");
+            _publicHashingService = new Mock<IPublicHashingService>();
+            _publicHashingService.Setup(x => x.HashValue(It.IsAny<long>())).Returns((long p) => $"CDE{p}");
```

Comment:
If the mocked hashed values are arbitrary, it would be nice to use "PUB" for public hashes and "PRI" for private ones to make the test code easier to scan.

Reply:
good idea